localstack.yaml

```
@@ -0,0 +1,204 @@
+
+subpackages:
+  - name: "${{package.name}}-compat"
+    description: "Compatibility package to place binaries in the location expected by upstream Dockerfile"
+    pipeline:
+      - name: Symlink scripts
+        runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/local/bin
+          for f in docker-entrypoint.sh docker-helper.sh localstack localstack-supervisor; do
+            ln -sf /usr/bin/$f "${{targets.contextdir}}"/usr/local/bin/$f
+          done
+      - name: Symlink binaries and libraries
+        runs: |
+          mkdir -p "${{targets.contextdir}}"/opt/code/localstack/bin
+          ln -sf /usr/share/localstack/.venv "${{targets.contextdir}}"/opt/code/localstack/.venv
+          for f in localstack localstack-supervisor; do
+            ln -sf /usr/bin/$f "${{targets.contextdir}}"/opt/code/localstack/bin/$f
+          done
+      - name: Symlink Java 11
+        runs: |
+          mkdir -p "${{targets.contextdir}}"/var/lib/localstack/lib/java/11
+          for f in bin conf legal lib release; do
+            ln -sf /usr/lib/jvm/java-11-openjdk/$f "${{targets.contextdir}}"/var/lib/localstack/lib/java/11/$f
+          done
+      - name: Symlink Python ${{vars.py-version}}
+        runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/local/lib
+          ln -sf /usr/lib/python${{vars.py-version}} "${{targets.contextdir}}"/usr/local/lib/python${{vars.py-version}}
+    test:
+      pipeline:
+        - name: tests for script symlinks
+          runs: |
+            for f in docker-entrypoint.sh docker-helper.sh localstack localstack-supervisor; do
+              stat /usr/local/bin/$f
+            done
+            for f in localstack localstack-supervisor; do
+              stat /opt/code/localstack/bin/$f
+            done
+            for f in bin conf legal lib release; do
+              stat /var/lib/localstack/lib/java/11/$f
+            done
+        - name: test symlinks
+          runs: |
+            stat /opt/code/localstack/.venv
+            stat /usr/local/lib/python${{vars.py-version}}
+            stat /usr/local/lib/python${{vars.py-version}}
+
+update:
+  enabled: true
+  github:
+    identifier: localstack/localstack
+    strip-prefix: v
+
+test:
+  environment:
+    contents:
+      packages:
+        - localstack-compat
+        - jq
+  pipeline:
+    - name: Check if Python venv properly installed
+      runs: |
+        source /opt/code/localstack/.venv/bin/activate
+        python3 -m localstack.runtime.init BOOT
+        localstack --version | grep ${{package.version}}
+        localstack -h
+        awslocal --version
+    - name: "start daemon"
+      uses: test/daemon-check-output
+      with:
+        start: docker-entrypoint.sh
+        timeout: 30
+        expected_output: |
+          LocalStack version
+          Ready
+    - name: do some curl tests
+      runs: |
+        source /opt/code/localstack/.venv/bin/activate
+        docker-entrypoint.sh &
+        sleep 3
```

```suggestion
        sleep 5
```

I'd feel safer with a slightly longer sleep or some looping around curl to make sure this is not flaky.

But not needed.